modules/serverless-create-domain-mapping-kn.adoc

```diff
@@ -0,0 +1,60 @@
+[id="serverless-create-domain-mapping-kn_{context}"]
+= Creating a custom domain mapping using the Knative CLI
```

Comment:
I've noticed that previous `kn service commands` sections has subtitle `Creating serverless applications using the kn CLI`. Personally I like "the Knative CLI" a bit more. Can we align those titles either way?

Reply:
Absolutely, I had been meaning to do this anyway. I've updated it where I've noticed it in modules but feel free to point out other places in future if you notice this isn't consistent. Thank you!